modules/olm-bundle-format.adoc

```diff
@@ -59,11 +59,19 @@ etcd
 === Optional objects
 
 The following objects can also be optionally included in the `/manifests`
-directory of a bundle:
+directory of a bundle. If relevant, support for specific API versions is also
+noted:
 
 .Supported optional objects
 * Secrets
 * ConfigMaps
+* PodDisruptionBudget
+** `policy/v1beta1`
+* PriorityClass
+** `scheduling.k8s.io/v1`
+** `scheduling.k8s.io/v1beta1`
+** `scheduling.k8s.io/v1alpha`
```

Comment:
I don't think we should state we support the alpha scheduling API. I think its actually more effort than its worth to keep this document up to date with the latest API versions (for example, if PDBs go to v1). WDYT about removing the API version info and just providing the object name. 

Technically, the way the code is written, it will support all API versions for the supported object regardless.